### PR TITLE
Slack API integration for MVP0

### PR DESCRIPTION
**Pull request by lennertjansen**

### CODE REVIEW

File: backend/app/flow_execution/integrations/slack.py [hunk 1]
```diff
@@ -59,32 +90,196 @@ def _model_to_query_params(self, model: BaseIntegrationActionModel) -> dict[str,
 
     def check_connectivity(self) -> bool:
         """
-        Checks if the Slack API is accessible and the SDK is working.
+        Checks if the Slack API is accessible by our application and the SDK is working.
```
Comment (orhanrauf) [inline backend/app/flow_execution/integrations/slack.py:93]: nice

File: requirements_20240326.txt
Comment (orhanrauf) [inline requirements_20240326.txt:1]: delete or git ignore

File: backend/local_slack.py
Comment (orhanrauf) [inline backend/local_slack.py:1]: delete or git ignore. let's keep only 1 template file for local testing

Comment (lennertjansen) [reply backend/local_slack.py:1]: will do.

File: requirements_20240326.txt
Comment (lennertjansen) [reply requirements_20240326.txt:1]: will do

**Pull request merged**